src/helpers.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use Symfony\Component\Process\Process;
+
+if (! function_exists('git_version')) {
+    /**
+     * Get latest git tag version.
+     *
+     * @return string
+     */
+    function git_version()
+    {
+        $process = new Process('git describe --tags $(git rev-list --tags --max-count=1)');
+        $process->run();
+        
+        return trim($process->getOutput()) ?: 'unreleased';
+    }
+}
+
+if (! function_exists('mkdir_if_not_exist')) {
+    /**
+     * Makes directory if not exists.
+     *
+     * @param  string  $dir
+     * @param  int  $mode
+     * @param  bool  $recursive
+     *
+     * @return void
+     */
+    function mkdir_if_not_exist($dir, $mode = 0700, $recursive = true)
```

Comment:
since this function is only relevant for creating hooks, I would prefer something like `create_hooks_dir`. That way, we would only need to pass in `$gitDir` at the call sites. We can all just remove the other arguments and hardcode in the `mkdir` call.

Reply:
Okay!